modules/govuk/manifests/node/s_db_admin.pp

```diff
@@ -26,8 +26,28 @@
   $postgres_password    = undef,
   $postgres_port        = '5432',
 ) {
-  include ::govuk::node::s_base
-  include govuk_env_sync
```

Comment:
Let's keep these here, it seems confusing to expect the `s_postgresql_db_admin_base` to provide these generic things - hopefully this is without consequence and puppet accepts that two places can define this.